src/Keboola/StorageApi/Client.php

```diff
@@ -1534,6 +1534,10 @@ private function uploadFileToS3(
             'version' => '2006-03-01',
             'retries' => $this->getAwsRetries(),
             'region' => $prepareResult['region'],
+            'http' => [
```

Comment:
preco nepridavas  este `'retries' => 40`

Reply:
retries sa deleguje s klienta takže sem to neměnil. Ikdyž když nad tým přemýšlám tak je to možná blbost.

Reply:
Upravil sem všude na 40